kubernetes/kubectl-startup.sh

```diff
@@ -19,7 +19,7 @@
 #
 
 echo "Setting Up Fineract service configuration..."
-kubectl create secret generic fineract-tenants-db-secret --from-literal=username=root --from-literal=password=$(head /dev/urandom | tr -dc A-Za-z0-9 | head -c 16)
+kubectl create secret generic fineract-tenants-db-secret --from-literal=username=root --from-literal=password=root
```

Comment:
Wait a second... doesn't this hardcode it to"root"?! That's..  probably not such a good idea? 😎

Reply:
Actually these usernames and passwords will be saved as secrets.  But anyway you can get those values by running the command `kubectl get secrets/fineract-tenants-db-secret --template={{.data.password}} | base64 --decode` . And the problem is that when we use a random password, after restarting the container we'll be getting access denied issue.

Reply:
I do have another solution for this. Not sure whether it'll help to sort out this issue. Let me try that way as well :)

Reply:
I have used an initcontainer instead of hardcoding the password. So before the container creation it will bind /var/lib/mysql directory which could help us to get rid of this "Access Denied" issue after restarting the container

Reply:
@vorburger Please review the new changes